scripts/start-console-k8s.sh

```diff
@@ -2,22 +2,28 @@
 
 set -euo pipefail
 
-# Set minikube IP and token (see scripts/configure-minikube.shcut)
-export BRIDGE_K8S_MODE_OFF_CLUSTER_ENDPOINT=${BRIDGE_K8S_MODE_OFF_CLUSTER_ENDPOINT:=https://$(minikube ip):8443}
+K8S_API_URL=$(kubectl config view -o jsonpath="{.clusters[?(@.name==\"$(kubectl config current-context)\")].cluster.server}")
```

Comment:
IMHO the static defaults are already good enough. If the user is switching between k8s clusters the line may yield unexpected results.

Reply:
the  static env vars will override this defaults, soe if you have `BRIDGE_K8S_MODE_OFF_CLUSTER_ENDPOINT`
defined it will take the value from there